docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "2"
+services:
+  web: 
+    build: .
+    ports: 
+      - 80:3000
+    command: bash -c 'while !</dev/tcp/db/5432; do sleep 1; done; npm run start:dev'
```

Comment:
1. is the while loop here explainable?
2. does it mean "hold port 5432 open in perpetuity for postgres?"

(just wondering, I think it's cool if this is a thing that people do)

Reply:
"While port 5432 on `db` is not active, sleep 1 s, when it is you can run the npm script".
Or "wait for postgres to be running"